.github/SECURITY.md

```diff
@@ -0,0 +1,23 @@
+<h2>Responsible disclosure policy</h2>
+
+At the IOTA Foundation, we consider the security of our systems a top priority. But no matter how much effort we put into system security, there can still be vulnerabilities present. If you've discovered a vulnerability, please follow the guidelines below to report it to our security team:
+<ul>
+	<li>E-mail your findings to security@iota.org. If the report contains highly sensitive information, please consider encrypting your findings using our contact@iota.org (466385BD0B40D9550F93C04746A440CCE5664A64) PGP key.</li>
+</ul>
+Please follow these rules when testing/reporting vulnerabilities:
+<ul>
+	<li>Do not take advantage of the vulnerability you have discovered, for example by downloading more data than is necessary to demonstrate the vulnerability.</li>
+	<li>Do not read, modify or delete data that you don't own.</li>
+	<li>We ask that you do not to disclosure the problem to third parties until it has been resolved.</li>
```

Comment:
Remove redundant `to`.

Reply:
Thanks 😄 I think we can just say `Do not disclose the problem to third parties until it has been resolved.`